code/modules/cargo/exports/sheets.dm

```diff
@@ -68,7 +68,7 @@
 	export_types = list(/obj/item/stack/sheet/rglass)
 
 /datum/export/stack/bscrystal
-	cost = 300
+	cost = 500
```

Comment:
Was 750 pre-bounty, 81denton requested them not to be nerfed as hard on the bounty PR. I agree. These are as rare as diamonds and production via xenobio is similiar to adamantine, but adamantine is worth a lot more.